libcodechecker/analyze/analysis_manager.py

```diff
@@ -566,6 +568,52 @@ def __create_timeout(analyzer_process):
                          (progress_checked_num.value, progress_actions.value,
                           action.analyzer_type, source_file_name))
 
+                # Remove the previously generated error file.
+                if os.path.exists(failed_dir):
+                    err_file = os.path.join(failed_dir, result_base + '.zip')
```

Comment:
What is this removing? Above we are already removing a zip file.

Reply:
I think a merge went wrong, this can be removed.